### PR TITLE
Feat: sum coinbase balance to Portfolio Balance

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: src/navigation/tabs/home/components/PortfolioBalance.tsx [hunk 1]
```diff
@@ -49,11 +50,26 @@ const PercentageContainer = styled.View`
...
     ({WALLET}: RootState) => WALLET.portfolioBalance,
   );
+
   const defaultAltCurrency = useAppSelector(({APP}) => APP.defaultAltCurrency);
 
+  // Sum only if same currency
+  var totalBalance: number = portfolioBalance.current;
+  if (
+    coinbaseUser &&
+    coinbaseUser.data.native_currency === defaultAltCurrency.isoCode
```
[the remark this answers is not in the document]
Comment (cmgustavo) [reply src/navigation/tabs/home/components/PortfolioBalance.tsx:68]: Good idea. I agree. Let me check if I can do a little change there.

File: src/navigation/tabs/home/components/PortfolioBalance.tsx [hunk 2]
```diff
@@ -49,11 +50,26 @@ const PercentageContainer = styled.View`
...
+  const coinbaseBalance =
+    useAppSelector(({COINBASE}) => COINBASE.balance[COINBASE_ENV]) || 0.0;
   const portfolioBalance = useSelector(
     ({WALLET}: RootState) => WALLET.portfolioBalance,
   );
+
   const defaultAltCurrency = useAppSelector(({APP}) => APP.defaultAltCurrency);
 
+  // Sum only if same currency
+  var totalBalance: number = portfolioBalance.current;
```
[the remark this answers is not in the document]
Comment (cmgustavo) [reply src/navigation/tabs/home/components/PortfolioBalance.tsx]: Done. thanks!

**Review by rastajpa — APPROVED** [review]
ACK

**Review by msalcala11 — APPROVED** [review]
LGTM

**Pull request merged**